src/index.html

```diff
@@ -22,6 +22,399 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
```

Comment:
The `href` attribute should lead to `#` since it's a logo link typically leading to the home page, and you're already on the index page. It's a common practice to make the logo link point to the home page (`href="#"`).